src/components/people/PersonPhoto/PersonPresenceIcon/index.tsx

```diff
@@ -8,32 +8,40 @@ import {
     CloseCircleIcon,
     ScheduleIcon,
     PhotoSize,
+    RemoveCircleIcon,
+    CircleIcon
 } from '@equinor/fusion-components';
 
 type PersonPresenceProps = {
-    presence: PersonPresenceAvailability;
+    presenceAvailability: PersonPresenceAvailability;
     size: PhotoSize;
 };
 
-const PersonPresenceIcon: React.FC<PersonPresenceProps> = ({ presence, size }) => {
+const PersonPresenceIcon: React.FC<PersonPresenceProps> = ({ presenceAvailability, size }) => {
     const presenceClasses = classNames(
         styles.presenceContainer,
         useComponentDisplayClassNames(styles),
         styles[size],
-        styles[presence.toLowerCase()]
+        styles[presenceAvailability?.toLowerCase()]
     );
 
     const presenceIcon = React.useMemo(() => {
-        if (presence === 'Online') {
+        if (presenceAvailability === 'Available' || 'AvailableIdle') {
```

Comment:
This probably dosnt work as it should, same issue on line 35
Needs to be similar to how it is on line 32